backend/drm/util.c

```diff
@@ -170,6 +170,30 @@ const char *conn_get_name(uint32_t type_id) {
 	}
 }
 
+static uint32_t get_fb_for_bo_legacy(struct gbm_bo *bo) {
+	struct gbm_device *gbm = gbm_bo_get_device(bo);
+
+	/* we only support this as a fallback of last resort for ARGB8888 visuals, like
+	 * xf86-video-modesetting does. */
+	if (gbm_bo_get_format(bo) == GBM_FORMAT_ARGB8888)
```

Comment:
Shouldn't this be `!= GBM_FORMAT_ARGB8888`?

Reply:
oh, yes.

Reply:
I pushed a fix for that typo, we're going to test it on a few G4 and G5 systems.  I'll update this PR once we've finished testing this version of the patch.

Reply:
Alright, everything looks good!